chia/wallet/wallet_state_manager.py

```diff
@@ -620,7 +620,9 @@ async def fetch_parent_and_check_for_cat(
                 cat_puzzle = construct_cat_puzzle(CAT_MOD, bytes32(bytes(tail_hash)[1:]), our_inner_puzzle)
                 if cat_puzzle.get_tree_hash() != coin_state.coin.puzzle_hash:
                     return None, None
-                if bytes(tail_hash).hex()[2:] in self.default_cats:
+                if bytes(tail_hash).hex()[2:] in self.default_cats or self.config.get(
```

Comment:
@Quexington please advise if `setdefault` would be more appropriate here

Reply:
No I think `get` is correct

Reply:
Okay, keeping get. `setdefault` would add it to the dictionary if it's missing.

I thought it would make sense that new versions of Chia populate the config with their new parameters, so the user is aware that they exist when they scroll through their config.yaml.